tyk-headless/templates/deployment-pmp.yaml

```diff
@@ -48,22 +46,12 @@ spec:
             drop:
               - all
         env:
-          # Lagacy support for Redis Cluster driver. Driver dropped in v3.0.0.
-          - name: REDIGOCLUSTER_SHARDCOUNT
-            value: "128"
-          {{- if .Values.mongo.enabled }}
-          - name: PMP_MONGO_MONGOURL
-            value: {{ include "tyk-headless.mongo_url" . | quote }}
-          - name: PMP_MONGO_MONGOUSESSL
-            value: "{{ default "false" .Values.mongo.useSSL }}"
-          - name: PMP_MONGOAGG_MONGOURL
-            valueFrom:
-              secretKeyRef:
-                name: {{ if .Values.secrets.useSecretName }} {{ .Values.secrets.useSecretName }} {{ else }} secrets-{{ include "tyk-headless.fullname" . }} {{ end}}
-                key: mongoURL
-          - name: PMP_MONGOAGG_MONGOUSESSL
-            value:  "{{ default "false" .Values.mongo.useSSL }}"
-          {{- end }}
+          - name: TYK_PMP_OMITCONFIGFILE
+            value: "true"
+          - name: TYK_PMP_ANALYTICSSTORAGETYPE
+            value: "redis"
+          - name: TYK_PMP_ANALYTICSSTORAGECONFIG_TYPE
```

Comment:
BTW, why do we need both TYK_PMP_ANALYTICSSTORAGECONFIG_TYPE and TYK_PMP_ANALYTICSSTORAGETYPE? 
isn't one deprecated

Reply:
I just translated the config files directly into environment variables. So this might not be the only variable. I did not want to modify the config so I don’t break anything 😆